diff --git a/dest/browser/crs/net_crs.js b/dest/browser/crs/net_crs.js
index 752d48255a066027de44bcc5867647a717711ece..f9395f00ddb82cdb3559ed153342f5cf51bdee21 100644
--- a/dest/browser/crs/net_crs.js
+++ b/dest/browser/crs/net_crs.js
@@ -65,7 +65,7 @@ export class NetCrs {
             return new Response(new Uint8Array([]));
         }
         const g1End = this.numPoints * 64 - 1;
-        return await retry(() => fetch('https://crs.aztec.network/g1.dat', {
+        return await retry(() => fetch('https://crs.aztec-cdn.foundation/g1.dat', {
             headers: {
                 Range: `bytes=0-${g1End}`,
             },
@@ -76,7 +76,7 @@ export class NetCrs {
      * Fetches the appropriate range of points from a remote source
      */
     async fetchG2Data() {
-        return await retry(() => fetch('https://crs.aztec.network/g2.dat', {
+        return await retry(() => fetch('https://crs.aztec-cdn.foundation/g2.dat', {
             cache: 'force-cache',
         }), makeBackoff([5, 5, 5]));
     }
@@ -125,7 +125,7 @@ export class NetGrumpkinCrs {
             return new Response(new Uint8Array([]));
         }
         const g1End = this.numPoints * 64 - 1;
-        return await fetch('https://crs.aztec.network/grumpkin_g1.dat', {
+        return await fetch('https://crs.aztec-cdn.foundation/grumpkin_g1.dat', {
             headers: {
                 Range: `bytes=0-${g1End}`,
             },
diff --git a/src/crs/net_crs.ts b/src/crs/net_crs.ts
index 51ce21ad57d8363323e6a14144bf3888e2937f45..4519a02eb205d6a14680552c9a26352e50293c3d 100644
--- a/src/crs/net_crs.ts
+++ b/src/crs/net_crs.ts
@@ -78,7 +78,7 @@ export class NetCrs {
     const g1End = this.numPoints * 64 - 1;
     return await retry(
       () =>
-        fetch('https://crs.aztec.network/g1.dat', {
+        fetch('https://crs.aztec-cdn.foundation/g1.dat', {
           headers: {
             Range: `bytes=0-${g1End}`,
           },
@@ -94,7 +94,7 @@ export class NetCrs {
   private async fetchG2Data(): Promise<Response> {
     return await retry(
       () =>
-        fetch('https://crs.aztec.network/g2.dat', {
+        fetch('https://crs.aztec-cdn.foundation/g2.dat', {
           cache: 'force-cache',
         }),
       makeBackoff([5, 5, 5]),
@@ -153,8 +153,7 @@ export class NetGrumpkinCrs {
     }
 
     const g1End = this.numPoints * 64 - 1;
-
-    return await fetch('https://crs.aztec.network/grumpkin_g1.dat', {
+    return await fetch('https://crs.aztec-cdn.foundation/grumpkin_g1.dat', {
       headers: {
         Range: `bytes=0-${g1End}`,
       },
